src/Raygun.NetCore.Tests.Blazor.WebAssembly/appsettings.json

```diff
@@ -1,4 +1,5 @@
 {
   "Raygun": {
+    "ApiKey": "APIKEY"
```

Comment:
Tests failed to initialize because the `appsettings.json` was missing this key and so the init method threw an exception.